src/index.js

```diff
@@ -302,6 +304,9 @@ export {
  * @prop {(address: Address) => Promise<TxInput[]>} getUtxos
  * Returns a complete list of UTxOs at a given address.
  *
+ * @prop {(utxoId: TxOutputId) => Promise<boolean>} hasUtxo;
```

Comment:
I think it will be cleaner for the doc-generation to remove the final `;`

Reply:
my automatic line-ending habit, hard at work : ) 

thanks, will fix

Reply:
fixed